index.js

```diff
@@ -45,7 +45,7 @@ function detectWowCommand(data) {
     'wow: command not found',
     'command not found: wow',
     'Unknown command \'wow\'',
-    '\'wow\' is not recognized.*'
+    '\'wow\' is not recognized'
```

Comment:
My Windows is in pt-br so the message is different, but maybe this chunk is the same everywhere?

```
ObjectNotFound: (wow:String)
```

Maybe it is a better string to check.

Reply:
I'm sorry that I'm not able to understand what you're proposing.
Are you saying let's check for `'wow' is not recognized.*` instead?

This is just a simplification - it's a regex match - so by default `.*` matches are matched. It's not necessary to add `.*` and that's the only reason I've removed. The removal of `.*` here should not affect the resulting regex matches.